characters/feats.py

```diff
@@ -15,16 +17,79 @@ def get_feats():
         feats.append(newFeat)
     return feats
 
+def get_feat_by_id(pk_id):
+    connection = characters_common.db_connection()
+    myCursor = connection.cursor()
+    myCursor.execute("SELECT \
+        pk_id, \
+        feat, \
+        prerequisites,\
+        description, \
+        author, \
+        created_at, \
+        private, \
+        nanite_cost, \
+        FROM feats WHERE pk_id=%s;" % pk_id)
```

Comment:
Scope problems with using pk_id as both a parameter variable and defined variable in this function?